### ISSUE TITLE
Questions about training data

### ISSUE THREAD

**Issue by luopeixiang** [zhawe01/fairseq-gec#15]
I found two fields in the training data, **source_label** and **target_label**,  each item consists of 0 and 1, and has a length equal to **src_tokens** and **target**.
Can you please explain the meaning of these two fields? How are they used during training？

**Comment by zhawe01** [comment]
We use the "edit-weight MLE", which gives different loss weights to the edited/un-changed tokens. So we need to know which tokens are edited. In our code, 0 means un-changed and 1 means edited.

**Comment by luopeixiang** [comment]
Thanks for your reply. By the way, the original paper mentioned multi-task learning, so, how do you choose the weight between these multi tasks? 
The code calculating model's loss in [cross_entropy.py](https://github.com/zhawe01/fairseq-gec/blob/master/fairseq/criterions/cross_entropy.py)  seems not include the  Token-level Labeling Task loss and  Sentence-level Copying Task.

**Comment by zhawe01** [comment]
Yes, that part of code was not included in this repository.
